lib/cli.js

```diff
@@ -89,6 +99,17 @@ function run(argv) {
 
   files = expandGlobs(files);
 
+  if (excludedFiles && excludedFiles.length) {
+    excludedFiles = expandGlobs(excludedFiles);
+    files = files.filter(function(file){
+      return excludedFiles.indexOf(file) === -1;
+    });
+    infoAboutExcludedFiles = 'files were excluded:\n';
+    infoAboutExcludedFiles += excludedFiles.join('\n');
+    infoAboutExcludedFiles += '\n';
+    exports.stdout.write(infoAboutExcludedFiles);
```

Comment:
we should always log extra information to `stderr`; `stdout` is reserved only for the actual output produced by the tool, that way we can easily compose multiple cli tools (eg. `cat foo.js | esformatter | grep 'bar'`)

I also think that this information is unnecessary (http://www.linfo.org/rule_of_silence.html)